guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+---
+Layout: post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#id-link-to-section)
+2. [Summary](#Summary)
+3. [First logical group of Page](#First)
```

Comment:
Needs a human readable name. "First logical group of Page" is not not sufficient. You can do stuff like Virtualbox setup, Ubuntu Setup, Establishing a Connection

Reply:
Also applies to 16 and 17.